File: aula2/Personagem.java
public class Personagem{
    String nome;
    int vida;
    int forca;

    int receberDano(int forca){
        return(vida = vida - forca);
    }
    boolean estaVivo(){
        if(vida > 0){
            return(true);
        } else{
            return(false);
        }
    }
    void ficha(){
        System.out.println(nome);
        System.out.println(vida);
        System.out.println(forca);
    }
    void atacar(Personagem alvo){
        alvo.receberDano(forca);
    }
}

File: aula2/personagem.py
class Personagem:
    def __init__(self, nome, vida, forca):
        self.nome = nome
        self.vida = vida
        self.forca = forca

    def receberDano(self, forca):
        self.vida = self.vida - forca

    def estaVivo(self):
        if self.vida > 0 :
            return 1
        else:
            return 0
        
    def ficha(self):
        print(self.nome)
        print(self.vida)
        print(self.forca)

    def atacar(self, alvo):
        alvo.receberDano(self.forca)